shared_model/backend/protobuf/block.hpp

```diff
@@ -56,7 +56,7 @@ namespace shared_model {
               return HashType(proto_->payload().prev_block_hash());
             }),
             signatures_([this] {
-              SignatureSetType sigs;
+              interface::SignatureSetType sigs;
```

Comment:
What was the problem? I mean why did you add a namespace?

Reply:
SignatureSetType was moved